src/api/app/models/kiwi/preference.rb

```diff
@@ -27,6 +27,7 @@ def editable?
 #  type_image                :integer
 #  type_containerconfig_name :string(255)
 #  type_containerconfig_tag  :string(255)
+#  version                   :string(255)
```

Comment:
[Nitpick] Do we need a validation for the version format?

>  The image version must be a three-part version number of the format: Major.Minor.Release.

Reply:
@adrianschroeter what do you think?